pkg/kyc/authorization.go

```diff
@@ -63,7 +63,7 @@ func (c *authorizationComponentMW) GetActions(ctx context.Context) ([]apikyc.Act
 	return c.next.GetActions(ctx)
 }
 
-func (c *authorizationComponentMW) GetUser(ctx context.Context, username string) (apikyc.UserRepresentation, error) {
+func (c *authorizationComponentMW) GetUserByName(ctx context.Context, username string) (apikyc.UserRepresentation, error) {
 	var action = KYCGetUser.String()
 	var targetRealm = c.realmName
 	var groupID = RegistrationOfficer
```

Comment:
We should call CheckAuthorizationOnTargetUser. 
groupID is the one of the target so it seems incorrect there.